tests/semanticdb/expect/InventedNames.expect.scala

```diff
@@ -22,10 +22,10 @@ given `* */*<-givens::InventedNames$package.`* *`.*/`: Long/*->scala::Long#*/ =
 given X with
 /*<-givens::InventedNames$package.given_X.*//*->givens::X#*/  def doX/*<-givens::InventedNames$package.given_X.doX().*/ = 7
 
-given (using X/*->givens::X#*/): Y/*->givens::Y#*/ with
+/*->givens::InventedNames$package.given_Y().(x$1)*/given /*<-givens::InventedNames$package.given_Y().*/(using X/*->givens::X#*//*<-givens::InventedNames$package.given_Y().(x$1)*/): Y/*->givens::Y#*/ with
   def doY/*<-givens::InventedNames$package.given_Y#doY().*/ = "7"
 
-given [T/*<-givens::InventedNames$package.given_Z_T#[T]*/]: Z/*->givens::Z#*/[T/*->givens::InventedNames$package.given_Z_T#[T]*/] with
+/*->givens::InventedNames$package.given_Z_T().[T]*/given /*<-givens::InventedNames$package.given_Z_T().*/[T/*<-givens::InventedNames$package.given_Z_T#[T]*//*<-givens::InventedNames$package.given_Z_T().[T]*/]: Z/*->givens::Z#*/[T/*->givens::InventedNames$package.given_Z_T#[T]*/] with
```

Comment:
we probably want to substitute any occurrence for `...given_Z_T#[T]` with `...given_Z_T().[T]`, and the span of `...given_Z_T().[T]` should be where `...given_Z_T#[T]` is